src/components/Equations.ts

```diff
@@ -108,10 +108,12 @@ export class Equations<TGameStartr extends FullScreenPokemon> extends Component<
         const ev: IValuePoints = this.newPokemonEVs();
```

Comment:
We're going to want to be able to create a new Pokemon with an item. I also don't want us to keep adding on to the list of parameters to this function (what if, say, you want to specify an item but not the moves?). Change this function to take in an object that has these properties on it.